sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/unix_time_helper.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Provides helper method for using unix time.
+ *
+ */
+
+#pragma once
+
+#include <azure/core/datetime.hpp>
+
+#include <chrono>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Common { namespace Internal {
```

Comment:
Get rid of Common namespace, put these in KeyVault::Internal

Reply:
There is an issue for that #1684. Let's fix all of those in that one